subgraph/mappings/helpers/vote-results.ts

```diff
@@ -67,8 +67,8 @@ export function loadProposalAndSaveVoteResults(
         );
 
         // assign voting data
-        vote.nbYes = voteResults.value4;
-        vote.nbNo = voteResults.value5;
+        vote.nbYes = voteResults.value3;
+        vote.nbNo = voteResults.value4;
```

Comment:
@sophiacodes just wanted to check if these fixes should be made to `master` branch too?

Reply:
Yup, they should and will do that today